pkg/registry/core/trace/registry_server.go

```diff
@@ -83,6 +83,10 @@ func (ts *traceServer) Recv() (*registry.NSERegistration, error) {
 	return reg, err
 }
 
+func (ts *traceServer) Context() context.Context {
```

Comment:
What is this new API for?

Reply:
It is grpc.ServerStream, and this method is required to proper Stream chaining.

Reply:
removed.